eventmesh-connectors/eventmesh-connector-http/src/main/resources/sink-config.yml

```diff
@@ -30,6 +30,6 @@ connectorConfig:
   port: 8987
   path: /test
   ssl: false
-  idleTimeout: 5
+  idleTimeout: 5000
   webhookConfig:
```

Comment:
Maybe you should leave a time unit.

Reply:
Leave a annotation? Provide unit selection? Or should it be kept consistent with `HTTPSourceConnecter`?

Reply:
just leave a comment telling it's in milis or secs~